modules/installation-special-config-rtkernel.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-rtkernel_{context}"]
+
+= Adding a real-time kernel to nodes (during installation)
+
+Some {product-title} workloads require a high degree of determinism.
+While Linux is not a real-time operating system, the Linux real-time
+kernel includes a preemptive scheduler that provides the operating
+system with real-time characteristics.
+
+If your {product-title} workloads require these real-time characteristics,
+you can set up your compute (worker) and/or master machines to use the
+Linux real-time kernel when you first install the cluster. To do this,
+create a MachineConfig object and inject that object into the set of manifest
+files used by Ignition during cluster setup, as described in the following
+procedure.
+
+[NOTE]
+====
+This procedure is fully supported with bare metal installations using
+systems that are certified for Red Hat Enterprise Linux for Real Time 8.
+Real time support in {product-title} is also limited to specific subscriptions.
```

Comment:
@ahardin-rh The feature also works on GCP and the procedure is also supported with GCP. So I suggest to add GCP in the NOTE